tests/jeos/prepare_firstboot.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable jeos-firstboot as required by openQA testsuite
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    # Login with default credentials (root:linux)
+    assert_screen('linux-login', 300);
+    type_string("root\n", wait_still_screen => 5);
+    type_string("linux\n", wait_still_screen => 5);
+
+    # Ensure YaST2-Firstboot is disabled, as we use jeos-firstboot in openQA
+    script_run("systemctl disable YaST2-Firstboot");
+
+    # Ensure jeos-firstboot is installed
+    zypper_call('in jeos-firstboot');
```

Comment:
At least in Leap15.1 we should also install 'snapper' and 'grub2-snapper-plugin' because we are using them in /usr/lib/jeos-firstboot and because their binaries are not present by default the jeos-firstboot service will fail and the machine goes off.

This should fix it:
```
+    # Install snapper tools called from /usr/lib/jeos-firstboot
+    zypper_call('in snapper grub2-snapper-plugin');

```
Anyway if those packages are already present in TW the zypper_call shouldn't fail.

Reply:
@thehejik which image do you use? JeOS-efi from Leap:15.1:ARM should have it.

Reply:
@ggardet currently this image https://build.opensuse.org/package/binary/download/openSUSE:Leap:15.1:ARM:Images:ToTest/JeOS:JeOS-raspberrypi3.aarch64/images/aarch64/openSUSE-Leap15.1-ARM-JeOS-raspberrypi3.aarch64-2019.02.15-Snapshot5.2.raw.xz

Reply:
`JeOS-raspberrypi3` has no btrfs partition, so no `snapper` or `grub2-snapper-plugin` installed by default. IMO, `jeos-firstboot` should not require them if there is no BTRFS partition.  It sounds like a `jeos-firstboot` bug to me. If it is a hard requirements, then it should have `Requires` for that in the spec file.

Reply:
you are right, I didn't expect it's not btrfs filesystem. 

@Vogtinator maybe we should make jeos-firstboot not depending on btrfs, WDYT?

Reply:
Should be possible - can you implement that, @gmoro?